wiki/Tournaments/ICC/2021/en.md

```diff
@@ -0,0 +1,277 @@
+    - Usage of any tools and programs that are against [osu!'s community rules](/wiki/Rules#community-rules) is strictly prohibited and will be straight up reported to the osu! team at will.
+
+### Tournament registration
+
+1. All interested players are required to register into the tournament individually.
+   - In order to be eligible to play in the tournament, each player is required to have the Indonesian flag displayed on their profile and a playcount of 3,000 or more in osu!catch.
+2. To ensure that all incoming registrations are serious and valid, every registered player will be checked in detail by the Tournament Management.
+3. The list of players who are deemed to be eligible to compete in the tournament will be published by the Tournament Management after the Registration phase has ended.
+4. The Tournament Management will only account for a maximum of 32 players to take part in the tournament.
+   - In the case where the number of registrants exceeds 32, the Tournament Management will only enlist 32 registrants with the highest osu!catch pp to participate in the tournament.
+5. Testplayers, referees, and map selectors may not participate as players in this tournament.
+
+### Group stage rules
+
+1. In the Group stage, all qualified players will be situated into different seeds based on considerations listed on [this table](https://cdn.discordapp.com/attachments/747506228577697973/884397385793564682/unknown.png). The players from each seed will then be drawn into eight groups at random based on the result of a livestreamed [random.org](http://www.random.org/lists/ "random.org") draw.
+2. All players in each group will compete against each other in a single multiplayer lobby using the Battle Royale system. In this system, all players will each in turn pick out two maps from the mappool to be played with the following turnouts after each map:
+   - 1st place: **+2 points**
+   - 2nd place: **+1 point**
+   - 3rd place: **+1 point**
+   - 4th place: **0 points**
+   - Winner of the Tiebreaker: **+1 point**
+3. Players are **not allowed** to ban maps in the Group stage.
+4. There is no Lower Bracket in the Group stage.
+5. Rankings of each group are determined by sorting the results of each individual player's performance in the following priority:
+   - Most points obtained.
+   - Winner of the Tiebreaker.
+   - Players who win by default.
+   - Whether there are players who are disqualified from the tournament.
+6. Two players who placed on top of their respective groups' rankings based on the criteria listed above will advance to the Knock-out stage.
+
+### Knock-out stage rules 
+
+1. The eight Group stage winners will be matched to the eight Group stage runners-up at random; that is, instead of following a predetermined match-up format (A1 vs H2, B1 vs G2, and so on), all the match-ups will be based on the result of a livestreamed [random.org](http://www.random.org/lists/ "random.org") draw.
```

Comment:
```suggestion
1. The eight Group stage winners will be matched to the eight Group stage runners-up at random; that is, instead of following a predetermined match-up format (A1 vs H2, B1 vs G2, and so on), all the match-ups will be based on the result of a livestreamed [random.org](http://www.random.org/lists/) draw.
```